src/Checkout.vue

```diff
@@ -70,8 +74,9 @@ export default {
               cancelUrl: this.cancelUrl,
               clientReferenceId: this.clientReferenceId,
               customerEmail: this.customerEmail,
-              items: this.items,
```

Comment:
The `items` option is still present in [the docs](https://stripe.com/docs/js/checkout/redirect_to_checkout#stripe_checkout_redirect_to_checkout-options-items). Please do not omit this.